zebra-scan/src/tests.rs

```diff
@@ -150,7 +139,7 @@ async fn scanning_from_fake_generated_blocks() -> Result<()> {
     // The fake block function will have our transaction and a random one.
     assert_eq!(cb.vtx.len(), 2);
 
-    let res = scan_block(
+    let res = zcash_client_backend::scanning::scan_block(
```

Comment:
Can we use the new function here?

Reply:
We need to create fake Zebra transactions that are worth scanning. I'm working on that. It's a bigger task than I thought.

Reply:
I started implementing it because I think having those transactions is worth it. I'm basically converting `CompactTx` to Zebra's `Transaction::V4`.

Reply:
Let's add more tests in another PR?

This PR is blocking some tickets, but the tests aren't.

Reply:
Yea, I'll submit the solution in another PR.